### PR TITLE
Mask access tokens

### PR DESCRIPTION
**Pull request by biilmann**
This replaces access tokens with ***** when sending instance configuration

### CODE REVIEW

File: conf/configuration.go [hunk 1]
```diff
@@ -48,6 +52,29 @@ type Configuration struct {
...
+}
+
+func githubEndpoint(endpoint string) string {
+	if endpoint == "" {
+		return DefaultGitHubEndpoint
+	}
+	return endpoint
+}
+
+func (gh *GitHubConfig) MarshalJSON() ([]byte, error) {
```
Comment (sgirones) [inline conf/configuration.go]: where is this used?

Comment (sgirones) [reply conf/configuration.go]: nevermind, found it

Comment (biilmann) [reply conf/configuration.go]: :)

**Comment by sgirones** [comment]
from @biilmann: "the main reason for masking is just to give an easy way to make sure the access token is set, vs empty"

**Comment by sgirones** [comment]
👍

**Pull request merged**